config.h.in

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define DXVK_NVAPI_BUILD_COMPILER "@BUILD_COMPILER@"
+#define DXVK_NVAPI_BUILD_COMPILER_VERSION "@BUILD_COMPILER_VERSION@"
+#define DXVK_NVAPI_BUILD_TARGET "@BUILD_TARGET@"
+#define DXVK_NVAPI_BUILD_TYPE "@BUILD_TYPE@"
```

Comment:
It's certainly more than I expected but I don't mind having all of it. Reproducibility shouldn't be hurt so :shipit: